apps/www/src/lib/plate/demo/plugins/autoformatIndentLists.ts

```diff
@@ -23,7 +23,8 @@ export const autoformatIndentLists: AutoformatRule[] = [
       toggleIndentList(editor, {
         listStyleType: ListStyleType.Decimal,
       }),
-    match: ['1. ', '1) '],
+    match: ['^\\d+\\.$ ', '^\\d+\\)$ '],
```

Comment:
Can we use RegExp objects here to avoid the need for extra backslashes?

Reply:
Sounds better but I am afraid it could involve a giant refactor
https://github.com/udecode/plate/blob/342ff1ea1ada47fdea1de368c764bc0802dbad4f/packages/autoformat/src/utils/getMatchRange.ts#L23-L25

Reply:
Here it does a slice to extract the last character as the trigger to match, and others are used to match. We cannot use regex to do that. A conversion regex -> string -> slice -> regex is still needed, or we need to refactor a lot.